bittide-instances/bin/Hitl.hs

```diff
@@ -0,0 +1,139 @@
+-- SPDX-FileCopyrightText: 2024 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+-- | Program that writes YAML configuration files to '_build/hitl', to be used
+-- by the TCL using Vivado to run hardware-in-the-loop tests.
+--
+-- By default, it writes all known files. If given an identifier, it will only
+-- write that one.
+module Main where
+
+import Prelude
+
+import Clash.Hitl (HitlTests, packAndEncode)
+import Control.Monad (forM_)
+import Data.List (intercalate)
+import Options.Applicative
+import Paths_bittide_instances (getDataFileName)
+import System.Directory (createDirectoryIfMissing)
+import System.Exit (die)
+import System.FilePath ((</>))
+
+import qualified Bittide.Instances.Hitl.BoardTest as BoardTest
+import qualified Bittide.Instances.Hitl.FincFdec as FincFdec
+import qualified Bittide.Instances.Hitl.FullMeshHwCc as FullMeshHwCc
+import qualified Bittide.Instances.Hitl.FullMeshSwCc as FullMeshSwCc
+import qualified Bittide.Instances.Hitl.SyncInSyncOut as SyncInSyncOut
+import qualified Bittide.Instances.Hitl.Transceivers as Transceivers
+import qualified Bittide.Instances.Hitl.VexRiscv as VexRiscv
+
+import qualified Bittide.Instances.Hitl.Tcl.ExtraProbes as ExtraProbes
+
```

Comment:
why

Reply:
Because it's an extra level deeper, I've mixed it in now